sdk/storage/azure-storage-queue/HISTORY.md

```diff
@@ -2,8 +2,37 @@
 
 ## Version 12.0.0b1:
 
-For release notes and more information please visit
-https://aka.ms/azure-sdk-preview1-python
+Version 12.0.0b1 is the first preview of our efforts to create a user-friendly and Pythonic client library for Azure Storage Queues. For more information about this, and preview releases of other Azure SDK libraries, please visit
+https://aka.ms/azure-sdk-preview1-python.
+
+**Breaking changes: New API desgin**
+- Operations are now scoped to a particular client:
+    - `QueueServiceClient`: This client handles account-level operations. This includes managing service properties and listing the queues within an account.
+    - `QueueClient`: The client handles operations within a particular queue. This includes creating or deleting that queue, as well as enqueueing and dequeueing messages.
+
+    These clients can be accessed by navigating down the client hierarchy, or instantiated directly using URLs to the resource (account or queue).
+    For full details on the new API, please see reference documentation.
+- New message iterator, for receiving messages from a queue in a continuous stream.
+- New underlying REST pipeline implementation, based on the new `azure-core` library.
+- Client and pipeline configuration is now available via keyword arguments at both the client level, and per-operation. See reference documentation for a full list of optional configuration arguments.
+- Authentication using `azure-identity` credentials
+  - see this package's
+  [documentation](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/keyvault/azure-keyvault-keys/README.md)
```

Comment:
Similarly, storage-queue's readme would be better here.

Reply:
Oops - copy+pasted a little too much ;)